### DISCUSSION TITLE
Can't authenticate on new Pop!_OS install

### DISCUSSION THREAD

**Discussion by cj2001** [community/community#8015] [Copilot Conversations] [answered]
I have just installed VS Code on my new Pop!_OS (21.04) install.  I have tried to install the Copilot Nightly install as well as the regular Copilot (yes, I am in the beta).  However, when I get the request to authenticate, I get the following error:

```
[ERROR] [default] [2021-11-23T00:44:52.452Z] GitHub Copilot could not connect to server. Extension activation failed: "undefined"
```

Any suggestions?

**Comment by davecheney** [comment, marked as answer]
@cj2001 please double check that you're using VSCode from https://code.visualstudio.com/download as some Linux distributions ship with Code OSS or VSCodium which Copilot does not currently support.

**Reply by cj2001** [reply]
Ah ha!  That appears to have fixed it!  I was installing from flatpak, but using the .deb made it work.  Thank you!
